### PR TITLE
Ensure embedded Play application started.

### PR DESCRIPTION
**Pull request by timothygordon32**
Service manager client has independent materializer.

### CODE REVIEW

File: src/main/scala/uk/gov/hmrc/play/it/servicemanager/ServiceManagerClient.scala [hunk 1]
```diff
@@ -16,23 +16,23 @@
...
+import uk.gov.hmrc.play.it.{ExternalService, TestId}
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.duration._
 import scala.concurrent.{Await, Future}
 
 
 object ServiceManagerClient {
 
-  implicit val mat: Materializer = Play.materializer
+  implicit val system = ActorSystem("serviceManagerClient")
```
Comment (abdusahin) [inline src/main/scala/uk/gov/hmrc/play/it/servicemanager/ServiceManagerClient.scala:35]: do we need to shutdown this later on?

File: src/main/scala/uk/gov/hmrc/play/it/MicroServiceEmbeddedServer.scala [hunk 2]
```diff
@@ -100,9 +99,10 @@ trait EmbeddedServiceOrchestrator extends ResourceProvider with StartAndStopServ
 
     val environment: Environment = Environment.simple(mode = applicationMode)
     val application: Application = new GuiceApplicationBuilder(environment = environment, configuration = config).build()
+    Play.start(application)
```
Comment (abdusahin) [inline src/main/scala/uk/gov/hmrc/play/it/MicroServiceEmbeddedServer.scala:102]: also do we need to call Play.stop tear down process ?

Comment (timothygordon32) [reply src/main/scala/uk/gov/hmrc/play/it/MicroServiceEmbeddedServer.scala:102]: `NettyServer` already takes care of this when it is stopped, which happens in `EmbeddedServiceOrchestrator#stop`

File: src/main/scala/uk/gov/hmrc/play/it/servicemanager/ServiceManagerClient.scala (hunk 1 above)
Comment (timothygordon32) [reply src/main/scala/uk/gov/hmrc/play/it/servicemanager/ServiceManagerClient.scala:35]: Shutting it down (below).

**Comment by abdusahin** [comment]
👍

**Pull request merged**